scheduler/pkg/agent/client.go

```diff
@@ -677,8 +679,9 @@ func (c *Client) sendModelEventError(
 	event agent.ModelEventMessage_Event,
 	err error,
 ) {
+	c.logger.WithError(err).Errorf("Failed to load model, sending error to scheduler")
```

Comment:
**explanation**: Extra verbose logging that was helpful to me when investigating an issue